internal/cloud/metadata_test.go

```diff
@@ -46,6 +46,14 @@ func dmidecodeNutanix() []byte {
 	`)
 }
 
+func systemdDetectVirtKVM() []byte {
+	return []byte("kvm")
+}
+
+func systemdDetectVirtEmpty() []byte {
+	return []byte("None")
```

Comment:
The default output is `none` (small case) XD

Reply:
true, thanks!